front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/FolderCardImageTooltip.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Looking at this component and the one below (`front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/FolderHeaderImageTooltip.tsx`), they share a lot in common save for the specific messages. We could pass these in as props and use component.